README.md

```diff
@@ -177,10 +177,20 @@ A shopcart list can be got with a `GET` request on `'/shopcarts'`. The response
     }
 ]
 ```
+## Updating a Shopcart Item
+A shopcart item can be updated with a `PUT` request on `'/shopcartitems/id'`.
+
+When an authorized user hits a PUT request on '/shopcartitems', the API will return the updated shopcart item with a status code 200. The response will be something like:
+
+```json
+{
+    "amount": 4,
+}
+```
```

Comment:
Would you mind clarifying that this JSON would be the content for the `PUT` request, not the response. The response contains the full shopcart item

Reply:
done!